ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/MiningConfiguration.java

```diff
@@ -213,7 +215,24 @@ public MutableInitValues getMutableInitValues() {
 
   @Value.Immutable
   public interface MutableInitValues {
-    Bytes DEFAULT_EXTRA_DATA = Bytes.EMPTY;
+    // Obtain the raw short version
+    String RAW_SHORT_VERSION = MutableInitValues.class.getPackage().getImplementationVersion();;
```

Comment:
I see why you are not able to use `BesuInfo` that is the best fit for this, so I am reworking a bit the code to be able to use it.

Reply:
Create this PR https://github.com/benaadams/besu/pull/1 against your repo